.github/workflows/SecurityScan.yml

```
@@ -22,12 +22,17 @@ jobs:
     - name: Checkout repository
-      uses: actions/checkout@v4
+      uses: actions/checkout@v7
+      with:
+        persist-credentials: false
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v3
+      uses: github/codeql-action/init@v4
       with:
         languages: ${{ matrix.language }}
+        persist-credentials: false
```

actionlint (1.7.12)

[error] input "persist-credentials" is not defined in action "github/codeql-action/init@v4". available inputs are "analysis-kinds", "build-mode", "check-run-id", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "quality-queries", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)